dist/demo-app/README.md

```diff
@@ -1,74 +1,27 @@
 # Capital Framework
 
-A Capital Framework demo project. Use grunt to process the source files in `src/` and output them to `dist/`:
+A Capital Framework sample project.
+To preview it, run `python -m SimpleHTTPServer` and open http://localhost:8000 in a web browser.
```

Comment:
Keeping in mind the type of user likely to download the precompiled archive, maybe also mention that they should just be able to open the `index.html` file from their file system in a browser?